packages/state-transition/src/cache/epochCache.ts

```diff
@@ -751,6 +793,32 @@ export class EpochCache {
     };
   }
 
+  /**
+   * Return indices of validators who attestested in `attestation`
+   */
+  getAttestingIndices(fork: ForkSeq, attestation: allForks.Attestation): number[] {
+    if (fork < ForkSeq.electra) {
+      const {aggregationBits, data} = attestation;
+      const validatorIndices = this.getBeaconCommittee(data.slot, data.index);
+
+      return aggregationBits.intersectValues(validatorIndices);
+    } else {
+      const {aggregationBits, committeeBits, data} = attestation as electra.Attestation;
+
+      // There is a naming conflict on the term `committeeIndices`
+      // In Lodestar it usually means a list of validator indices of participants in a committee
+      // In the spec it means a list of committee indices according to committeeBits
+      // This `committeeIndices` refers to the latter
+      const committeeIndices = committeeBits.getTrueBitIndexes();
+
+      const validatorIndices = this.getBeaconCommittees(data.slot, committeeIndices);
+
+      const attestingIndices = new Set(aggregationBits.intersectValues(validatorIndices));
```

Comment:
return the array from `intersectValues` directly, there should not be any duplicates here
if there is, it'll fail the validation anyway, the created `Set` does not help